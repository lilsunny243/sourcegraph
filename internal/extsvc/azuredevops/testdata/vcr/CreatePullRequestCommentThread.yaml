---
version: 1
interactions:
- request:
    body: '{"Comments":[{"parentCommitId":0,"content":"new comment","commentType":1}]}'
    form: {}
    headers:
      Content-Type:
      - application/json
    url: https://dev.azure.com/sgtestazure/sgtestazure/_apis/git/repositories/sgtestazure3/pullrequests/5/threads?api-version=7.0
    method: POST
  response:
    body: '{"pullRequestThreadContext":null,"id":27,"publishedDate":"2023-02-17T01:31:49.473Z","lastUpdatedDate":"2023-02-17T01:31:49.473Z","comments":[{"id":1,"parentCommentId":0,"author":{"displayName":"idan.varsano@sourcegraph.com","url":"https://spsprodcca1.vssps.visualstudio.com/Ab768986f-7304-43dd-be74-f37e30f66c38/_apis/Identities/473dec3e-03d7-6147-b106-0b0a7f766a92","_links":{"avatar":{"href":"https://dev.azure.com/sgtestazure/_apis/GraphProfile/MemberAvatars/aad.NDczZGVjM2UtMDNkNy03MTQ3LWIxMDYtMGIwYTdmNzY2YTky"}},"id":"473dec3e-03d7-6147-b106-0b0a7f766a92","uniqueName":"idan.varsano@sourcegraph.com","imageUrl":"https://dev.azure.com/sgtestazure/_apis/GraphProfile/MemberAvatars/aad.NDczZGVjM2UtMDNkNy03MTQ3LWIxMDYtMGIwYTdmNzY2YTky","descriptor":"aad.NDczZGVjM2UtMDNkNy03MTQ3LWIxMDYtMGIwYTdmNzY2YTky"},"content":"new
      comment","publishedDate":"2023-02-17T01:31:49.473Z","lastUpdatedDate":"2023-02-17T01:31:49.473Z","lastContentUpdatedDate":"2023-02-17T01:31:49.473Z","commentType":"text","_links":{"self":{"href":"https://dev.azure.com/sgtestazure/_apis/git/repositories/601cf461-9338-4396-895d-a3e2780bd52f/pullRequests/5/threads/27/comments/1"},"repository":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/601cf461-9338-4396-895d-a3e2780bd52f"},"threads":{"href":"https://dev.azure.com/sgtestazure/_apis/git/repositories/601cf461-9338-4396-895d-a3e2780bd52f/pullRequests/5/threads/27"},"pullRequests":{"href":"https://dev.azure.com/sgtestazure/_apis/git/pullRequests/5"}}}],"threadContext":null,"properties":null,"identities":null,"isDeleted":false,"_links":{"self":{"href":"https://dev.azure.com/sgtestazure/_apis/git/repositories/601cf461-9338-4396-895d-a3e2780bd52f/pullRequests/5/threads/27"},"repository":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/601cf461-9338-4396-895d-a3e2780bd52f"}}}'
    headers:
      Access-Control-Expose-Headers:
      - Request-Context
      Activityid:
      - 8382071e-80a5-4a37-bc64-f3b6835b2086
      Cache-Control:
      - no-cache, no-store, must-revalidate
      Content-Type:
      - application/json; charset=utf-8; api-version=7.0
      Date:
      - Fri, 17 Feb 2023 01:31:48 GMT
      Expires:
      - "-1"
      P3p:
      - CP="CAO DSP COR ADMa DEV CONo TELo CUR PSA PSD TAI IVDo OUR SAMi BUS DEM NAV
        STA UNI COM INT PHY ONL FIN PUR LOC CNT"
      Pragma:
      - no-cache
      Request-Context:
      - appId=cid-v1:72d31d95-1757-44f5-b910-a46611808454
      Strict-Transport-Security:
      - max-age=31536000; includeSubDomains
      Vary:
      - Accept-Encoding
      X-Cache:
      - CONFIG_NOCACHE
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - SAMEORIGIN
      X-Msedge-Ref:
      - 'Ref A: 5848E542FDE5480E856F8C54039F3CD9 Ref B: YTO01EDGE0719 Ref C: 2023-02-17T01:31:49Z'
      X-Tfs-Processid:
      - b406c152-6c95-4bc1-9596-68cb4d8f6e3b
      X-Tfs-Session:
      - 8382071e-80a5-4a37-bc64-f3b6835b2086
      X-Vss-E2eid:
      - 8382071e-80a5-4a37-bc64-f3b6835b2086
      X-Vss-Senderdeploymentid:
      - 4ff21e82-8865-0b2e-ffe8-9598818f8190
      X-Vss-Userdata:
      - 473dec3e-03d7-6147-b106-0b0a7f766a92:idan.varsano@sourcegraph.com
    status: 200 OK
    code: 200
    duration: ""
