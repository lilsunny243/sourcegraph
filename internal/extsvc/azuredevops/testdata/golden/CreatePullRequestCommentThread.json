{
  "id": 27,
  "Comments": [
   {
    "id": 1,
    "publishedDate": "2023-02-17T01:31:49.473Z",
    "lastUpdatedOn": "0001-01-01T00:00:00Z",
    "content": "new comment",
    "commentType": "text"
   }
  ],
  "publishedDate": "2023-02-17T01:31:49.473Z",
  "lastUpdatedOn": "0001-01-01T00:00:00Z",
  "isDeleted": false
 }